netlify.toml

```diff
@@ -0,0 +1,6 @@
+[build.environment]
+  NODE_VERSION = "16"
```

Comment:
Better to put the node version in an `.nvmrc` file so you're local setup also knows which version to use